po/vi.po

```diff
@@ -3345,85 +3345,106 @@ msgstr ""
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid "Blocks"
-msgstr ""
+msgstr "Blocks (khối lệnh)"
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid ""
 "A block in Rust contains a sequence of expressions, enclosed by braces `{}`. "
 "Each block has a value and a type, which are those of the last expression of "
 "the block:"
 msgstr ""
+"Mỗi block trong Rust chứa một chuỗi những biểu thức, được bao bọc bởi dấu `{}"
+"`. Mỗi block có một giá trị và một kiểu dữ liệu, và là biểu thức cuối cùng "
+"của block đó:"
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid "\"y: {y}\""
-msgstr ""
+msgstr "\"y: {y}\""
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid ""
 "If the last expression ends with `;`, then the resulting value and type is "
 "`()`."
 msgstr ""
+"Nếu biểu thức cuối cùng kết thúc bằng dấu `;`, giá trị trả về và kiểu dữ "
+"liệu sẽ là `()`."
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid ""
 "You can show how the value of the block changes by changing the last line in "
 "the block. For instance, adding/removing a semicolon or using a `return`."
 msgstr ""
+"Giáo viên có thể trình bày rằng giá trị của block thay đổi bằng cách thay "
+"đổi dòng cuối cùng của block đó. Ví dụ, thêm vào / loại bỏ dấu `;` hoặc sử "
+"dụng lệnh `return`"
 
 #: src/control-flow-basics/blocks-and-scopes/scopes.md
 msgid "A variable's scope is limited to the enclosing block."
 msgstr ""
+"Mỗi phạm vi của một biến đều được giới hạn trong một block (khối lệnh)."
```

Comment:
I think we should retain the original meaning of `scope` instead of translating it to `phạm vi`. This makes it clearer that we are referring to a Rust's term, which implies the following: `The variable is destroy when the program go outside the block`